spec/decanter/decanter_core_spec.rb

```diff
@@ -596,6 +596,21 @@ def self.name
       end
     end
 
+    context 'with missing args keys with a :default_value in the decanter' do
```

Comment:
I think we should include test cases for:
- Not overriding an existing value
- Not overriding a `nil` value
- Not overriding a `blank?` value

Reply:
@chawes13  it seems like this is expected current behavior based on other specs, but sending an empty string should result in a value of `nil` rather than passing the empty string right?

Reply:
Exactly. An empty string should return `nil` instead of the default value provided